### ISSUE TITLE
Zendesk Support - missing entries, does not sync all ticket_metrics

### ISSUE THREAD

**Issue by jablonskijakub** [airbytehq/airbyte#13951]
## Environment
- **Airbyte version**:  0.36.4 alpha
- **OS Version / Instance**: AWS EC2
- **Deployment**: Docker
- **Source Connector and version**: Zendesk Support 0.2.10
- **Destination Connector and version**: Postgres via S3
- **Step where error happened**: Sync job 

## Current Behavior
When I sync `ticket_metrics` I can only fetch a subset of all the data and I am missing lots of rows even though my set up is as below:

![image|690x139](upload://nY2mIlz6fdYXa56dSWfxXnp6NjV.png)

What I noticed is that there is no tickets which were closed in 2021 and earlier, also `updated_at` is mostly 2022-MM-DD

![image|392x500](upload://etBGoxziNgO52YarCS0uu0FuXLb.png)

This is the view I have in Zendesk for 2021 
![image|690x389](upload://qf76IhZ4awijLS8mqVqVo138yhy.png)

[logs-853.txt|attachment](upload://F08MxudWiL8iGiiPuQRPRhTgPG.txt) (33.3 KB)

## Expected Behaviour
It is possible to sync all `ticket_metrics`

## Are you willing to submit a PR?
No

**Comment by alafanechere** [comment]
Hey @jablonskijakub is it a behavior you only observe on the `ticket_metrics` stream?

**Comment by jablonskijakub** [comment]
yes, I haven't noticed anything suspicious in other streams `(satisfaction_ratings, tickets, ticket_audits, ticket_metric_events`)

**Comment by jablonskijakub** [comment]
hey. apparently there was a similar issue in other streams and it was fixed by this PR. maybe it helps here as well

https://github.com/airbytehq/airbyte/pull/12122
